VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+
+* **Year 0:** 100
+* **Year 1:** 113
+* **Year 2:** 127
+* **Year 3:** 144
+* **Year 4:** 163
+* **Year 5:** 184
+* **Year 6:** 208
+* **Year 7:** 235
+* **Year 8:** 265
+* **Year 9:** 300
+* **Year 10:** 300
+
+### Is there a minimum amount of ATOMs that must be staked to be a validator?
+
+There is no minimum. The top 100 validator candidates with the highest total stake (where total stake = self-bonded stake + delegators stake) are the one that become validators. 
+
+### How will delegators choose their validators?
+
+Delegators are free to choose validators according to their own subjective criteria. However, there are a number of objective criteria that can be expected to influence delegators decision when it comes to choosing their validators, namely:
+* **Commission rate:** Validators with low commissions will likely be more attractive.
+* **Amount of self-bonded ATOMs:** Number of ATOMs a validator self-bonded to its staking pool. A validator with higher amount of self-bonded ATOMs has more skin-in-the-game, making it more liable for its actions.
+* **Amount of delegated ATOMs:** Total number of ATOMs delegated to a validator. A high stake shows that the community trusts this validator, but it also means that this validator is a bigger target for hackers. Indeed, hackers are incentivized to hack bigger validators as they receive a reward proportionate to the stake of the validator they can prove to have compromised. We expect validators to become less and less attractive as their amount of delegated ATOMs grows.
+* **Track record:** Delegators will likely look at the track record of the validators they plan to delegate to. This includes seniority, past votes on proposals, historical average uptime and how often the node was compromised.
+
+Apart from these objective criteria that will be displayed in the Tendermint wallet’s UI, there will be a possibility for validators to signal a website address to complete their resume. Validators will need to build reputation one way or another to attract delegators. For example, it would be a good practice for validators to have their setup audited by third parties. Note though that the Tendermint company will not approve or conduct any audit itself.
+
+### Do validators need to be publicly identified?
+
+No, they don’t. Each delegator will value each validator based on objective and subjective criteria. Validators will be able (and are recommended) to register a website address when they nominate themselves so that they can advertise their operation as they see fit. Some delegators may prefer a website that clearly displays the team running the validator and their resume, while others might prefer anonymous validators with positive track records. Most likely both identified and anonymous validators will coexist in the validator set.
+
+### What are the directives of a validator?
```

Comment:
Let's call it `responsibilities` instead of `directives`. Being a validator is a burden.

Reply:
addressed